packages/salesforcedx-vscode-apex/src/views/testOutlineProvider.ts

```diff
@@ -246,6 +246,9 @@ export abstract class TestNode extends vscode.TreeItem {
         dark: DARK_ORANGE_BUTTON
       };
     }
+
+    const contextValue = this.contextValue.split('_');
+    this.contextValue = `${contextValue}_${outcome}`;
```

Comment:
Changed the name of this method to highlight it's doing more than updating an icon now. I'm appending the outcome of a test to the contextValue of the test node so i can access this in the commands' where clauses as defined in the package.json.